python/sglang/srt/_custom_ops.py

```diff
@@ -0,0 +1,118 @@
+# Adapted from https://github.com/vllm-project/vllm/blob/a6221a144af772fd1a68fe7e627935dc53e81738/vllm/_custom_ops.py
```

Comment:
If this is exactly a version release commit, can you use something like
https://github.com/vllm-project/vllm/blob/v0.6.4.post1/vllm/_custom_ops.py